dev/AppNotifications/ShellLocalization.h

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "pch.h"
+
+// In this file we define methods that write/return Shell assets, like DisplayName and icon URI.
+
+namespace Microsoft::Windows::AppNotifications::ShellLocalization
+{
+    const std::wstring c_appIdentifierPath{ LR"(Software\Classes\AppUserModelId\)" };
+    const std::wstring c_quote{ LR"(")" };
+    const std::wstring c_backSlash{ LR"(\)" };
+    const std::wstring c_localMicrosoftFolder{ LR"(\Microsoft)" };
+    const std::wstring c_localWindowsAppSDKFolder{ LR"(\WindowsAppSDK)" };
+    const std::wstring c_pngExtension{ LR"(.png)" };
```

Comment:
Why `wstring`? `wstring` is a deep copy. Malloc+memcpy+free, when it's a compile-time known constant.

For string constants like these you should use `PCWSTR`


You can use `wstring_view` in some cases but I can never remember the right syntax to get equally zero-runtime compile-time construction. `constexpr std::wstring_view foo{ L"bar" };` ?